glide.yaml

```diff
@@ -72,3 +72,14 @@ import:
   subpackages:
   - compute/metadata
   - internal
+- package: k8s.io/kubernetes
+  version: e4a1ca7105fa9a3075fbf13a3286f8dddd59a658
```

Comment:
Changing that import let me run glide. So I'll try to update imports and see if it compiles.